virtual_rainforest/main.py

```diff
@@ -333,6 +244,13 @@ def vr_run(
     # Container to store paths to continuous data files
     continuous_data_files = []
 
+    # Only variables in the data object that are updated by a model should be output
+    all_variables = [
+        models_cfd[model_nm].vars_updated for model_nm in models_cfd.keys()
+    ]
+    # Then flatten the list to generate list of variables to output
+    variables_to_save = [item for sublist in all_variables for item in sublist]
```

Comment:
You can use `itertools` for this:

```suggestion
    variables_to_save = list(chain.from_iterable(all_variables))
```

You'd need to have `from itertools import chain` somewhere for this.